### PR TITLE
Send H5 format result file, code, and vcd file of the specific RID

### LINKED ISSUE #39: Send the specific H5 format result file
### Feature you want to implement
Send the specific H5 format result file (corresponding to the received RID).

### How the feature is implemented
When receiving a request with the url, `~/result/{RID}/h5/`, it will return the corresponding H5 format result file.

### Additional context
About requesting a file using `FastAPI`, see [FastAPI document](https://fastapi.tiangolo.com/advanced/custom-response/?#fileresponse).

### LINKED ISSUE #57: Send the specific code and vcd file
### Feature you want to implement
Send the specific experiment code and vcd file in the received RID directory.

### How the feature is implemented
When receiving a request with the url, `~/result/{RID}/code/` or `~/result/{RID}/vcd`, it will return the corresponding file wrapped by [`FileResponse`](https://fastapi.tiangolo.com/advanced/custom-response/?#fileresponse).

### Additional context
The implementation will be similar to #39.

### PR DESCRIPTION
**Pull request by BECATRUE**
This closes #39 and closes #57.

Implementing to send these three files is similar, so I implemented these in one function, i.e. `get_result()`.

If there is no valid file, it would return 404 error so I didn't handle it explictily.

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -431,6 +433,32 @@ async def list_result_directory() -> List[int]:
     return rid_list
 
 
+class ResultFileType(str, Enum):
+    h5 = "h5"
+    code = "code"
+    vcd = "vcd"
```
Comment (kangz12345) [inline main.py]: Please use UPPERCASE as pylint complains.

File: main.py [hunk 2]
```diff
@@ -431,6 +433,32 @@ async def list_result_directory() -> List[int]:
...
+
+
+@app.get("/result/{rid}/{result_file_type}/")
+async def get_result(rid: str, result_file_type: ResultFileType) -> FileResponse:
+    """Gets the H5 format result file of the given RID and returns it.
```
Comment (kangz12345) [inline main.py]: The returning file is "H5 format result file" only if the `result_file_type` is h5, right?

**Review by kangz12345 — CHANGES_REQUESTED** [review]

File: main.py (hunk 1 above)
Comment (BECATRUE) [reply main.py]: Oh, I'm sorry that I didn't check the Pylint complains. I will fix it.

File: main.py (hunk 2 above)
Comment (BECATRUE) [reply main.py]: Yes😂 I didn't update the docstring..

**Review by kangz12345 — APPROVED** [review]
LGTM

**Pull request merged**